backend/pyproject.toml

```diff
@@ -160,7 +161,7 @@ dependencies = [
     "django-cors-headers<4.0.0,>=3.13.0",
     "django-countries<8.0.0,>=7.3.2",
     "django-csp<4.0,>=3.7",
-    "django-elasticsearch-dsl<8.0.0,>=7.2.2",
+    "django-elasticsearch-dsl==8.0.0",
```

Comment:
do we want to have <9.0 or <8.1? @KonradMarzec1991

Reply:
The same like above, 8.1